contracts/finance/PaymentSplitter.sol

```diff
@@ -92,6 +92,13 @@ contract PaymentSplitter is Context {
         return _erc20TotalReleased[token];
     }
 
+    /**
+     * @dev Getter for the amount of payees.
+     */
+    function totalPayees() public view returns (uint256) {
+        return _payees.length;
+    }
+
```

Comment:
Also, as I said before (somehow my comment was missed), this new function should go just after `function payee(uint256)`.

Function ordering is important for readability / auditability